libs/barista-components/table/src/table.html

```diff
@@ -5,6 +5,7 @@
 
 <ng-template #emptyStateTemplate>
   <ng-content
+    *ngIf="!loading"
```

Comment:
I am fine with this but it's worth pointing out that this only works because we don't have a default ng-content in this template. So I guess at some point we might stumble across this issue again.